### PR TITLE
Feature/type adjustments

### PR DESCRIPTION
**Pull request by niklasl**
See individual commits. These *should* be uncontroversial and agreed upon.

### CODE REVIEW

File: source/vocab/things.ttl [hunk 1]
```diff
@@ -57,8 +57,9 @@
     rdfs:comment "Abstrakt basklass för :Instance och :Item."@sv;
     rdfs:subClassOf :Endeavour .
 
-:Hub a owl:Class ; # LC Uses in conjunction with Work. SubClass a WorkHub and keep as baseClass in KBV?
-    # rdfs:label "Hub"@en, "Nav"@sv ;  - TODO: Naming things...
+:WorkHub a owl:Class ;
+    rdfs:label "Work Hub"@en, "Verkshubb"@sv ;  # TODO: Naming things...
```
Comment (olovy) [inline source/vocab/things.ttl:61]:
```suggestion
    rdfs:label "Work Hub"@en, "Verksnav"@sv ;  # TODO: Naming things...
```

**Review by olovy — APPROVED** [review]
LGTM
See comment

**Review by klngwll — APPROVED** [review]
Is Classification decided? (referring to general discussions on making things less precise but still usable in different context with skos:Concept).

Not sure why we made DataDownload an `equivalentClass` in the first place...

File: source/vocab/things.ttl [hunk 2]
```diff
@@ -57,8 +57,9 @@
...
+    skos:altLabel "Verksnav"@sv ;
```
Comment (niklasl) [inline source/vocab/things.ttl:62]:
```suggestion
    skos:altLabel "Verkshubb"@sv ;
```

**Comment by niklasl** [comment]
`Classification` is decided insofar as these are not to be used as `subject` but as `classification` (in Swepub). So `Topic` is not desired as the type. De-typing all `Classification` terms as `Concept` is another challenge.

**Comment by niklasl** [comment]
I'm going to keep "hubb" for now; we'll see in tests what crops up (perhaps hubs won't fly anyway).

**Pull request merged**